lib/pronto/bitbucket_server.rb

```diff
@@ -0,0 +1,34 @@
+module Pronto
+  class BitbucketServer < Bitbucket
+    def pull_comments(sha)
+      @comment_cache["#{pull_id}/#{sha}"] ||= begin
+        client.pull_comments(slug, pull_id).map do |comment|
+          if comment['commentAnchor']
+            Comment.new(sha, comment['comment']['text'], comment['commentAnchor']['path'], comment['commentAnchor']['line'])
+          else
+            Comment.new(sha, comment['comment']['text'])
```

Comment:
Why do we need to access properties via `['comment']['text']`?
What would it take to make it more similar to `bitbucket.rb`, accessing it like `comment.text`?